ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master.yaml

```diff
@@ -21,6 +21,9 @@ images:
       as:
       - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6
   to: csi-driver-manila
+- dockerfile_path: images/cinder-csi-plugin/Dockerfile
+  from: base
+  to: openstack-cinder-csi-driver
```

Comment:
The name isn't consistent with the manila CSI image: `openstack-cinder-csi-driver` vs `csi-driver-manila`.
What's the preferred naming scheme, and is there a plan to update the manila image name in case we want it to be `<project>-csi-driver`?

Reply:
We standardized the naming pattern recently. All drivers should be called as <platform>-<name>-csi-driver, and their operators as <platform>-<name>-csi-driver-operator.

Examples: https://github.com/openshift/aws-ebs-csi-driver-operator and https://github.com/openshift/aws-ebs-csi-driver

https://github.com/openshift/ovirt-csi-driver-operator and https://github.com/openshift/ovirt-csi-driver

According to this pattern the operator is called https://github.com/openshift/openstack-cinder-csi-driver-operator and the driver name should be openstack-cinder-csi-driver

Reply:
Are there plans to update the manila CSI image to adhere to the naming convention?